modules/openapi-generator/src/main/resources/kotlin-client/libraries/jvm-okhttp/infrastructure/ApiClient.kt.mustache

```diff
@@ -168,6 +170,15 @@ import org.threeten.bp.OffsetTime
         if (bodyContent.isEmpty()) {
             return null
         }
+        if (T::class.java == File::class.java) {
+            // return tempfile
+            val f = createTempFile()
```

Comment:
Can we use `Files.createTempFile()` instead of `File.createTempFile()` has security issue?

ref: https://cwe.mitre.org/data/definitions/378.html

Reply:
Added via https://github.com/OpenAPITools/openapi-generator/pull/9284